app/Http/Controllers/BookController.php

```diff
@@ -36,11 +36,12 @@ public function __construct(EntityRepo $entityRepo, UserRepo $userRepo, ExportSe
      */
     public function index()
     {
-        $books = $this->entityRepo->getAllPaginated('book', 10);
+        $books = $this->entityRepo->getAllPaginated('book', 16);
```

Comment:
Changing of **10** to **16** books per page will effect in list grid also. 
I can understand your intention to made it as 16 ( 4 grid per row ), but it should not affect the list grid.